### PR TITLE
#181 Add HAR2Script convertor

### PR DESCRIPTION
**Pull request by songeunwoo**
- add template file.
  - add upload_har_modal.ftl.
  - add HAR convertor

### CODE REVIEW

File: ngrinder-controller/src/main/java/org/ngrinder/script/controller/FileEntryController.java [hunk 1]
```diff
@@ -532,4 +533,29 @@ public int compare(FileEntry o1, FileEntry o2) {
...
+	 * @return Map resultMap
+	 */
+	@RequestMapping(value = "/convert/file", method = RequestMethod.POST)
+	@ResponseBody
+	public Map<String, Object> convertToFile(@RequestParam("uploadFile") MultipartFile file) throws IOException, JSONException {
+		return fileEntryService.convertToFile(file);
+	}
+
+	/**
+	 * Convert textarea string with a NGrinder Trmplate.
```
Comment (junoyoon) [inline ngrinder-controller/src/main/java/org/ngrinder/script/controller/FileEntryController.java]: Trmplate ==> tempate

File: ngrinder-controller/src/main/java/org/ngrinder/script/handler/ScriptHandler.java [hunk 2]
```diff
@@ -297,6 +301,22 @@ public String getScriptTemplate(Map<String, Object> values) {
 		}
 	}
 
+	/**
+	 * return Quarter of a nGrinder template file name.
```
Comment (junoyoon) [inline ngrinder-controller/src/main/java/org/ngrinder/script/handler/ScriptHandler.java]: What it is?

**Comment by songeunwoo** [comment]
@junoyoon I'll fix it again commit.

File: ngrinder-controller/src/main/webapp/WEB-INF/ftl/script/editor.ftl [hunk 3]
```diff
@@ -224,7 +223,7 @@
 					validating = false;
 					$('#validation_result_pre_div').text(res);
 					$('#validation_result_panel').show();
-					$('#script_samples_link').hide();
+					$('#script_create_helper').hide();
```
Comment (junoyoon) [inline ngrinder-controller/src/main/webapp/WEB-INF/ftl/script/editor.ftl]: script_create_helper? script_generation_helper.

File: ngrinder-controller/src/main/webapp/WEB-INF/ftl/script/editor.ftl [hunk 4]
```diff
@@ -248,8 +247,21 @@
 					editor.setSize(null, 500);
 				}
 			});
-		});
 
+			$('#uploadHAR2ScriptBtn').click(function() {
```
Comment (junoyoon) [inline ngrinder-controller/src/main/webapp/WEB-INF/ftl/script/editor.ftl]: upload_har2script_btn

**Comment by junoyoon** [comment]
Use a-b-c format when specifying id in html.

**Pull request merged**